packages/scandipwa/src/component/Popup/Popup.style.scss

```diff
@@ -102,6 +102,18 @@
             position: fixed;
         }
     }
+
+    .NotificationList {
+
+        @include desktop {
+            display: none;
+        }
```

Comment:
Since the notification should be rendered only on mobile devices, please check the `isMobile` prop and render it only if it's true.